src/containers/Toolbar/index.tsx

```diff
@@ -12,6 +12,8 @@ interface IToolbarProps {
   onPreserveLogsChange: (preserveLogs: boolean) => void;
   inverted: boolean;
   onInvertedChange: (isInverted: boolean) => void;
+  isRegexActive: boolean;
+  setIsRegexActive: (isRegexActive: boolean) => void;
```

Comment:
For react props could we drop the `is` and use `regexActive` and `onRegexActiveChange`

Reply:
sure thing!